app/models/manageiq/providers/base_manager/event_catcher.rb

```diff
@@ -32,5 +32,16 @@ def self.validate_config_settings(configuration = VMDB::Config.new("vmdb"))
 
       subclasses.each { |subclass_key| configuration.merge_from_template_if_missing(*(path + [subclass_key])) }
     end
+
+    validate_event_flooding_settings(configuration, :defaults)
+  end
+
+  def self.validate_event_flooding_settings(config, provider)
```

Comment:
ooops, and it looks like this validate stuff has gone away too, since config revamp :(